compass/ocean/tests/sphere_transport/correlated_tracers_2d/forward.py

```diff
@@ -0,0 +1,101 @@
+import time
+
+from compass.model import run_model
+from compass.step import Step
+from datetime import timedelta
+
+
+class Forward(Step):
+    """
+    A step for performing forward MPAS-Ocean runs as part of the
+      correlated_tracers_2d.
+
+    Attributes
+    ----------
+    resolution : int
+        The resolution of the (uniform) mesh in km
+    """
+
+    def __init__(self, test_case, resolution, dt_minutes):
+        """
+        Create a new step
+
+        Parameters
+        ----------
+        test_case :
+            compass.ocean.tests.global_convergence.correlated_tracers_2d.CorrelatedTracers2D
```

Comment:
This is one of 2 exceptions (the other being long URLs) to the line length limit.  This docstring won't format correctly when it is parsed, so it needs to be one line. Sorry about that!